acbuild/acbuild.go

```diff
@@ -239,3 +262,30 @@ func stdout(format string, a ...interface{}) {
 	out := fmt.Sprintf(format, a...)
 	fmt.Fprintln(os.Stdout, strings.TrimSuffix(out, "\n"))
 }
+
+func addACBuildAnnotation(a *lib.ACBuild, cmd *cobra.Command, args []string) error {
+	if a == nil {
```

Comment:
Oh. I had intended to call it with non-nil when the `--modify` flag was used, but after reviewing a thing or two it's unnecessary. Removed the argument.